lib/src/main/java/io/ably/lib/http/HttpCore.java

```diff
@@ -20,12 +20,8 @@
 import io.ably.lib.rest.Auth;
 import io.ably.lib.transport.Defaults;
 import io.ably.lib.transport.Hosts;
-import io.ably.lib.types.AblyException;
-import io.ably.lib.types.ClientOptions;
-import io.ably.lib.types.ErrorInfo;
-import io.ably.lib.types.ErrorResponse;
-import io.ably.lib.types.Param;
-import io.ably.lib.types.ProxyOptions;
+import io.ably.lib.types.*;
```

Comment:
I would expect this to fail checkstyle as we've got the `AvoidStarImport` rule enabled. Our preference is to explicitly import required types. I know IDEs can have different opinions on this, but for us it's about code clarity and reducing namespace pollution.

Reply:
Changed it back to how it was https://github.com/ably/ably-java/pull/671/commits/9b683325229128f70ab53335012647e206978262